src/components/Container/Container.vue

```diff
@@ -31,11 +31,13 @@ export default {
   methods: {
     formatTemp (location) {
       const temperature = location.current.weather.tp
-      return parseFloat((temperature * 9 / 5 + 32).toFixed(1))
+      const fahrenheitTemp = parseFloat((temperature * 9 / 5 + 32).toFixed(1))
+      return fahrenheitTemp
```

Comment:
I also really like this refactor so that we return a single variable rather than an equation. I think this follows some best-practice principles really well.

Reply:
Agreed, perfect name too - love how clear this is.